x/ibc-hooks/contracts/crosschain-swaps/Cargo.toml

```diff
@@ -0,0 +1,68 @@
+# This feature has been stabilized in 1.64. If using an earlier version (like in
+# rust-optimizer-arm:0.12.8) you should uncomment this like to be able to
+# compile the contract
```

Comment:
I generally think we should declare an MSRV of Rust 1.64 (two versions behind latest)

Reply:
Why not latest? I find nightly annoying cause it changing so often means mental overhead, but since the code is compiled to wasm, any version should work. AFAIK there are no issues installing the versions to get the code to compile.

Reply:
I feel like it depends on how much of a library this is supposed to be. When I was an active rust dev, I often found it super annoying when dependencies would declare latest MSRV, when I wanted to provide longer term guarantees, or have things not break at every rust release. (Dependents would complain about latest MSRV)

(Also this is not remotely PR blocking, or even release blocking)